client/blocks/reader-share/index.jsx

```diff
@@ -198,51 +193,49 @@ class ReaderShare extends React.Component {
 					</span>
 				</span>,
 				this.state.showingMenu &&
-					( canShareToWordPress()
-						? <SitesPopover
-								key="menu"
-								header={
-									<div>
-										{ this.props.translate( 'Share on:' ) }
-									</div>
-								}
-								context={ this.refs && this.refs.shareButton }
-								visible={ this.state.showingMenu }
-								groups={ true }
-								onSiteSelect={ this.pickSiteToShareTo }
-								onClose={ this.closeMenu }
-								position={ this.props.position }
-								className="reader-share__sites-popover"
-							/>
-						: <ReaderPopoverMenu
-								key="menu"
-								context={ this.refs && this.refs.shareButton }
-								isVisible={ this.state.showingMenu }
-								onClose={ this.closeExternalShareMenu }
-								position={ this.props.position }
-								className="popover reader-share__popover"
-								popoverTitle={ translate( 'Share on' ) }
-							>
-								<PopoverMenuItem
-									action="twitter"
-									className="reader-share__popover-item"
-									title={ translate( 'Share on Twitter' ) }
-								>
-									<SocialLogo icon="twitter" />
-									<span>Twitter</span>
-								</PopoverMenuItem>
-								<PopoverMenuItem
-									action="facebook"
-									className="reader-share__popover-item"
-									title={ translate( 'Share on Facebook' ) }
-								>
-									<SocialLogo icon="facebook" />
-									<span>Facebook</span>
-								</PopoverMenuItem>
-							</ReaderPopoverMenu> ),
+					<ReaderPopoverMenu
+						key="menu"
+						context={ this.refs && this.refs.shareButton }
+						isVisible={ this.state.showingMenu }
+						onClose={ this.closeExternalShareMenu }
+						position={ this.props.position }
+						className="popover reader-share__popover"
+						popoverTitle={ translate( 'Share on' ) }
+					>
+						<PopoverMenuItem
+							action="facebook"
+							className="reader-share__popover-item"
+							title={ translate( 'Share on Facebook' ) }
+						>
+							<SocialLogo icon="facebook" />
+							<span>Facebook</span>
+						</PopoverMenuItem>
+						<PopoverMenuItem
+							action="twitter"
+							className="reader-share__popover-item"
+							title={ translate( 'Share on Twitter' ) }
+						>
+							<SocialLogo icon="twitter" />
+							<span>Twitter</span>
+						</PopoverMenuItem>
+						{ this.props.hasSites
```

Comment:
this is a pretty meh way of expressing the conditional render.
lets switch it from:
```jsx
this.props.hasSites ? <SitesSelector ... /> : null
``` 
to
```jsx
this.props.hasSites && <SiteSelector .... />
```